data/djangocon-2015/videos/building-theatlantic-com-homepages-wysiwyg-admin.json

```diff
@@ -1,7 +1,7 @@
 {
   "category": "",
   "slug": "building-theatlantic-com-homepages-wysiwyg-admin",
-  "title": "Building theatlantic.com homepage\u2019s WYSIWYG admin with... by Frankie Dintino",
+  "title": "Building theatlantic.com homepage’s WYSIWYG admin with Django and Knockout",
```

Comment:
Actually, it's probably better to use `\u2019` in the JSON (I copy/pasted the title from the web page, without checking non-ASCII characters).
I'll fix this